download_models.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
could you move this to forecasts? Or perhaps into scripts? This keeps the top level nice and clear. 

Could you also add what model this downloads? and when do you need to use this script?